src/libConsensus/ConsensusBackup.cpp

```diff
@@ -672,8 +673,8 @@ bool ConsensusBackup::ProcessMessageCollectiveSigCore(const vector<unsigned char
     }
 
     // N-byte bitmap
-    m_responseMap = GetBitVector(collectivesig, curr_offset, GetBitVectorLengthInBytes(m_pubKeys.size()));
-    curr_offset += GetBitVectorLengthInBytes(m_pubKeys.size()) + 2;
+    m_responseMap = BitVector::GetBitVector(collectivesig, curr_offset, BitVector::GetBitVectorLengthInBytes(m_pubKeys.size()));
+    curr_offset += BitVector::GetBitVectorLengthInBytes(m_pubKeys.size()) + 2;
```

Comment:
The same refactoring could happen here (e.g., `BitVector::GetBitVectorSerializedSize(m_pubKeys.size())`).

Reply:
Agree. Pushed update.